docs/superpowers/plans/2026-04-26-agent-wait-process-boundary-hardening.md

```
@@ -0,0 +1,702 @@
+and then:
+
+```json
+{"name": "bash_process", "arguments": {"action": "logs", "job_id": "<job_id>", "tail": 200}}
+```
+
+Use `sleep_and_wait(wake_type="timer", delay_seconds=...)` only when the agent intentionally wants to pause and wake itself later. A timer wait does not monitor bash process completion.
+
+## Browser CLI Upstream Boundary
+
+A `browser-cli status` value such as `degraded` with missing trusted workspace binding is browser-cli runtime state. Agiwo should report that status and preserve logs, but should not invent browser-cli-specific recovery semantics inside scheduler wait handling.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)